raco/backends/myria/myria.py

```diff
@@ -1872,6 +1882,7 @@ def fire(self, expr):
     rules.OneToOne(algebra.UnionAll, MyriaUnionAll),
     rules.OneToOne(algebra.Difference, MyriaDifference),
     rules.OneToOne(algebra.OrderBy, MyriaInMemoryOrderBy),
+    rules.OneToOne(algebra.Limit, MyriaLimit),
```

Comment:
I tried switching the CollectBeforeLimit rule to use the 'algebra.Limit' operator rule, but it leads to infinite recursion when applying the rules. It seems like applying the physical operator was purposely used as a stopping condition (so the rules don't apply to the new local limit operator we generate).